addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -30,6 +30,8 @@ $font-family-sans-serif: $o-theme-font !default;
 $headings-font-family: $o-theme-headings-font !default;
 $headings-color: palette-color('h1') !default;
 
+$lead-font-size: 1.125rem;
```

Comment:
Don't understand this commit. Also we recently remove the use of the lead class because FP find it confusing and not possible to add through the editor... -> other PR with more explanation @Cocographique

Reply:
**PR** [46902](https://github.com/odoo/odoo/pull/46902)